OpenSim/Simulation/Test/testContactGeometry.cpp

```diff
@@ -549,7 +549,7 @@ void testIntermediateFrames() {
 
         // Scaffolding for the ball.
         auto* linkOffset = new PhysicalOffsetFrame("link_offset",
-                model.updBodySet().get("point"),
+                model.getBodySet().get("point"),
```

Comment:
could replace this with `model.getComponent<Body>("point")` since you are updating anyways